src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidget.tsx

```diff
@@ -168,6 +173,16 @@ const DetailsWidget = ({
           </div>
         </div>
       )}
+      {detailsForAction.Permissions && (
+        <div className={styles.item}>
+          <div className={styles.label}>
+            <FormattedMessage {...MSG.roles} />
```

Comment:
The "Roles" label should be aligned to the top of the section, not the middle _(see the same way we do it for domains descriptions)_

![Screenshot from 2021-02-10 17-37-31](https://user-images.githubusercontent.com/1193222/107533424-98bb2e80-6bc7-11eb-9dee-251e9a628e8b.png)

Everything else here is fine _(padding between the roles lables)_